index.html

```diff
@@ -17,10 +17,44 @@
   </head>
   <body>
     <header>
-      <div class="container"></div>
+      <div class="container">
+        <nav>Code Journal</nav>
+      </div>
     </header>
     <main class="container">
-      <div data-view="entry-form"></div>
+      <h1>New Entry</h1>
+      <div data-view="entry-form">
+        <form>
+          <div class="row">
+            <img
+              class="column-full column-half"
+              src="images/placeholder-image-square.jpg"
+              alt="a placeholder-image-square" />
```

Comment:
Rather than putting the `column` classes on the `img`, I would wrap the `img` inside a `div` and apply the classes there. It's common practice to wrap `img`'s inside of a `div` and control the column size on the `div`. This way it makes it easier to scale the image size.

Don't forget to add `width: 100%` to your `img` after doing this